### ISSUE TITLE
Bookmarks column

### ISSUE THREAD

**Issue by UjCbFwtBayFM** [dimdenGD/OldTweetDeck#140]
#119 added the option to add tweets to bookmarks, but they can't be viewed from Tweetdeck.... I know it's a longshot but would it be possible to add this feature?

**Comment by spideymang** [comment]
Yes please it would be fantastic to have access to the Column https://twitter.com/i/bookmarks and see all the saved bookmarks

**Comment by dimdenGD** [comment]
Done
